TestsProject/TestsProjectTests/hOCR.m

```diff
@@ -0,0 +1,61 @@
+//
+//  hOCR.m
```

Comment:
It closely related to recognition(it is recognition). You could create ```context``` for hOCR. Don't think that there many tests are needed. Any way it is better them all to lay in one place for now.